search-service/src/main/kotlin/com/egm/stellio/search/listener/EntityListener.kt

```diff
@@ -41,24 +41,31 @@ class EntityListener(
             EventType.UPDATE -> {
                 // TODO add missing checks:
                 //  - existence of temporal entity attribute
-                //  - only add an attribute instance if observedAt (?)
                 //  - needs optimization (lot of JSON-LD parsing, ...)
                 val rawParsedData = jacksonObjectMapper().readTree(entityEvent.payload!!)
                 val rawEntity = parseEntity(entityEvent.updatedEntity!!)
                 val attributeName = rawParsedData.fieldNames().next()
-                val rawAttributeValue = rawParsedData[attributeName]["value"]
+                val attributeValuesNode = rawParsedData[attributeName]
+
+                if (!attributeValuesNode.has("observedAt")) {
+                    logger.debug("Ignoring update event for $attributeName, it has no observedAt information")
```

Comment:
Should be at least info or warning as it has functional value ? (any user could be interested in that part, right?)

Reply:
Yes, INFO level makes sense (not WARNING, because it is not an unexpected scenario)